pkg/proc/eval.go

```diff
@@ -1828,7 +1828,39 @@ func (v *Variable) findMethod(mname string) (*Variable, error) {
 		}
 		return r, nil
 	}
+	return v.tryFindMethodInEmbeddedFileds(mname)
+}
 
+func (v *Variable) tryFindMethodInEmbeddedFileds(mname string) (*Variable, error) {
+	structVar := v.maybeDereference()
+	structVar.Name = v.Name
+	if structVar.Unreadable != nil {
+		return structVar, nil
+	}
+	switch t := structVar.RealType.(type) {
+	case *godwarf.StructType:
+		for _, field := range t.Field {
+			isEmbeddedStructMember :=
```

Comment:
This code was necessary to support older versions of Go, just use `field.Embedded` directly, i.e. remove `isEmbeddedStructMember` and use field.Embedded in its place.